sdk/eventhub/event-hubs/src/eventHubReceiver.ts

```diff
@@ -617,21 +615,18 @@ export class EventHubReceiver extends LinkEntity {
    * @hidden
    */
   private _createReceiverOptions(options: CreateReceiverOptions): RheaReceiverOptions {
-    if (options.newName) this.name = uuid();
     const rcvrOptions: RheaReceiverOptions = {
       name: this.name,
       autoaccept: true,
       source: {
         address: this.address
       },
       credit_window: 0,
-      onMessage: options.onMessage || ((context: EventContext) => this._onAmqpMessage(context)),
-      onError: options.onError || ((context: EventContext) => this._onAmqpError(context)),
-      onClose: options.onClose || ((context: EventContext) => this._onAmqpClose(context)),
-      onSessionError:
-        options.onSessionError || ((context: EventContext) => this._onAmqpSessionError(context)),
-      onSessionClose:
-        options.onSessionClose || ((context: EventContext) => this._onAmqpSessionClose(context))
```

Comment:
Nit: Alternatively, you can update `_createReceiverOptions` to take only the event position. That will allow us to get rid of the interface `CreateReceiverOptions`